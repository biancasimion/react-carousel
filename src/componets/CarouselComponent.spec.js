import React from 'react';
import Carousel from './CarouselComponent';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Carousel />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});