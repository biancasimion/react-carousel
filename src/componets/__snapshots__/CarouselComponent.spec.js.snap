// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="carousel"
  >
    <button
      className="button prev"
      onClick={[Function]}
    >
      Previous
    </button>
    <button
      className="button next"
      onClick={[Function]}
    >
      Next
    </button>
    <div
      className="slides active-slide-index-undefined"
    >
      <div
        className="slides__container"
        style={
          Object {
            "transform": "translateX(-NaN%)",
          }
        }
      />
    </div>
  </div>
</div>
`;
